ignition_ros2_control/src/ignition_ros2_control_plugin.cpp

```diff
@@ -36,6 +36,7 @@
 #include <string>
 #include <utility>
 #include <vector>
+#include <boost/shared_ptr.hpp>
```

Comment:
uhmm, is this required?

Reply:
This include was needed for me in order to pass compilation because the boost library is used [here](https://github.com/wyca-robotics/ign_ros2_control/blob/57032614eda91e59d375f6bdbe46790582f95042/ignition_ros2_control/src/ignition_ros2_control_plugin.cpp#L83-L84)

Reply:
I removed the boost dependency, you should be able to remove this header now.